frontend/src/main/components/SectionsInstructorTableBase.js

```diff
@@ -0,0 +1,78 @@
+import React, { Fragment } from "react";
+import { useTable, useGroupBy, useExpanded } from "react-table";
+import { Table } from "react-bootstrap";
+
+// Stryker disable StringLiteral, ArrayDeclaration
+export default function SectionsInstructorTableBase({
+  columns,
+  data,
+  testid = "testid",
+}) {
+  // Stryker disable next-line ObjectLiteral
+  const { getTableProps, getTableBodyProps, headerGroups, rows, prepareRow } =
+    useTable(
+      {
+        initialState: {
+          groupBy: [],
+          hiddenColumns: ["isSection"],
```

Comment:
If the column is always hidden, do we really need it?

Reply:
@pconrad I use the isSection column to see if I need to style a given row like a lecture (dark blue and bolded), or like a section (light blue and non-bold text). Is that ok to do?

https://github.com/ucsb-cs156-f23/proj-courses-f23-7pm-3/blob/a5161081e8891d1a8483c26e0300455cd19b5c59/frontend/src/main/components/SectionsInstructorTableBase.js#L56